utils/gear-hooks/src/context/Api.tsx

```diff
@@ -1,33 +1,90 @@
-import { createContext, useEffect, useState } from 'react';
 import { GearApi } from '@gear-js/api';
+import { WsProvider, ScProvider } from '@polkadot/api';
+import * as Sc from '@substrate/connect';
+import { createContext, useEffect, useMemo, useRef, useState } from 'react';
+
 import { ProviderProps } from 'types';
-import { WsProvider } from '@polkadot/api';
 
-type Value = {
-  api: GearApi;
-  isApiReady: boolean;
+type WsProviderArgs = {
+  endpoint: string | string[];
+  autoConnectMs?: number | false;
+  headers?: Record<string, string>;
+  timeout?: number;
+};
+
+type ScProviderArgs = {
+  spec: string | Sc.WellKnownChain;
```

Comment:
Is wellknownchain necessary here?

Reply:
Absolutely not, yeah, I guess it's misleading since we only need Gear endpoints.